src/components/GWL.vue

```diff
@@ -460,8 +569,9 @@ export default {
           .attr("transform", "translate(90, 0)")
                   
         var legend_keys = ["Very low", "Low", "Normal", "High","Very high"]; // labels
+        var legend_color = [this.verylow, this.low, this.normal,this.normal, this.high, this.veryhigh];
         var shape_dist = [5,25,42,42,60,83,103]; // y positioning (normal has 2 shapes butted together)
```

Comment:
New version:
![image](https://user-images.githubusercontent.com/17803537/139607746-3af6251c-1534-4d62-9c57-4b6a4305bdff.png)

I like that the glyphs show the movement of the peaks, but what I don't like is that we are binning the data into 5 uneven groups and not showing that in some way.

Reply:
Also cleaned this up so Normal is a single glyph and can be coded in a much cleaner way

Reply:
Hmmm, that's a good point. Could we incorporate a histogram somehow, to show the distribution? I also like the idea of labeling the breakpoints, rather than the individual ranges, so there's less repeated text.

Reply:
How would you do that here with the breakpoints?

Reply:
Here's a blurry ugly rough sketch, but I think you could label the breakpoints like so (w/ or w/o the histogram):
![IMG-4641](https://user-images.githubusercontent.com/54007288/139691446-0f839202-492d-4624-a55c-52b8fe344e36.jpg)